deploy/kustomize/testing/controller-coverage-patch.yaml

```diff
@@ -14,23 +14,3 @@
     hostPath:
       path: /var/lib/pmem-csi-coverage
       type: DirectoryOrCreate
-# driver in controller mode runs as non-root user
-# which cannot have enough permissions to access
-# coverage-dir(/var/lib/pmem-csi-coverage)
-# Change the ownership of this folder to "pmem-csi:pmem-csi"
-# so that driver could write coverage reports.
-- op: add
-  path: /spec/template/spec/initContainers
```

Comment:
Seems like this change(removing init container) is not related to this commit?

Reply:
It is. Because we now run as root (at least where we test), we don't need this workaround for non-root anymore.